database.go

```diff
@@ -11,14 +11,29 @@ import (
 
 var db *sql.DB
 
+func checkErrTx(err error, tx *sql.Tx) {
```

Comment:
Please remove this function

Reply:
Ok, if I remove this function, have I to use the newer function closeTransaction() instead of it in all those places in which are used?

Reply:
Yes, of course! But please in a different PR